regression/syscall_test/blocklists/dup_test

```diff
@@ -0,0 +1 @@
+DupTest.DupClearsCloExec
```

Comment:
This test needs a syscall eventfd which is not implemented, so it is blocked.

Reply:
We need improve current `eventfd` implementation.

Reply:
Good news. Eventfd has been merged.

Reply:
Could you remove this test from the block list?

Reply:
Even with `eventfd`, it still has some panics. I am working on it.

Reply:
I finally find the error from `EventFile` in `eventfd`. Now it has been fixed.